projects/plugins/boost/app/lib/class-speed-score-request.php

```diff
@@ -196,18 +196,20 @@ public function poll_update() {
 		);
 
 		if ( is_wp_error( $response ) ) {
+			// Special case: If the request is not found, restart it.
+			if ( 'not_found' === $response->get_error_code() ) {
+				return $this->restart();
```

Comment:
Can this cause an infinite loop of restarts? Or is that the intention?

Reply:
19c3d99727 adds a limit to how often it's allowed to retry.